en/simulation/gazebo_vehicles.md

```diff
@@ -27,6 +27,23 @@ make px4_sitl gazebo
 make px4_sitl gazebo_iris_opt_flow
 ```
 
+<a id="quadrotor_depth_camera"></a>
```

Comment:
FYI only you only need an explicit anchor if you are linking to the section from somewhere AND you expect the title is likely to change at some point. I tend to do this for longer headings that get linked a lot of times. But it is better to use the auto-generated heading anchor if you can because when you press the link navigation works better.

```suggestion
```